todoapp/build.gradle

```diff
@@ -1,9 +1,11 @@
 buildscript {
     repositories {
+        // As of Gradle plugin 3.0 you can use the `google()` shortcut instead of `maven { url 'https://maven.google.com' }`
+        google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.1'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta5'
```

Comment:
We don't use beta versions... but since you've already made all the changes and stable might happen soon, it's ok.

Reply:
I thought it was going to happen sooner, so I wanted to make sure we're ready.